lib/db/mysql.js

```diff
@@ -752,6 +753,59 @@ module.exports = function (log, error) {
   }
 
 
+  // USER EMAILS
+  // Insert : emails
+  // Values : normalizedEmail = $1, email = $2, uid = $3, emailCode = $4, isVerified = $5, isPrimary = $6, verifiedAt = $7, createdAt = $8
+  var CREATE_EMAIL = 'CALL createEmail_1(?, ?, ?, ?, ?, ?, ?, ?)'
+  MySql.prototype.createEmail = function (uid, data) {
+    return this.write(
+      CREATE_EMAIL,
+      [
+        data.normalizedEmail,
+        data.email,
+        uid,
+        data.emailCode,
+        data.isVerified,
+        data.isPrimary,
+        data.verifiedAt,
+        data.createdAt
+      ]
+    )
+  }
+
+  // Select : emails
+  // Values : uid = $1
+  var ACCOUNT_EMAILS = 'CALL accountEmails_1(?)'
+  MySql.prototype.accountEmails = function (uid) {
+    return this.readOneFromFirstResult(
```

Comment:
Yea, believe it is. The name is kind of confusing to what it is actually doing, but it is returning `result[0]` similar to [securityEvents](https://github.com/mozilla/fxa-auth-db-mysql/blob/master/lib/db/mysql.js#L1087).